docs/setup_config.rst

```diff
@@ -0,0 +1,40 @@
+Setup configuration
+===================
+
+Loading notification configuration from a YAML file
+***************************************************
+
+This library provides a ``ConfigurationStep``
+(from the library ``django-setup-configuration``, see the
+`documentation <https://github.com/maykinmedia/django-setup-configuration>`_
+for more information on how to run ``setup_configuration``)
+to configure the notification configuration.
+
+To add this step to your configuration steps, add ``django_setup_configuration`` to ``INSTALLED_APPS`` and add the following setting:
+
+    .. code:: python
+
+        SETUP_CONFIGURATION_STEPS = [
+            ...
+            "notifications_api_common.contrib.setup_configuration.steps.NotificationConfigurationStep"
+            ...
+        ]
+
+The YAML file that is passed to ``setup_configuration`` must set the
+``notifications_config_enable`` flag to ``true`` to enable the step. All fields under ``notifications_config`` are optional.
+
+Example file:
+
+    .. code:: yaml
+
+        notifications_config_enable: True
+        notifications_config:
+          notifications_api_service_identifier: notifs-api
```

Comment:
I think it would make the documentation more complete if we also mention somewhere that this points to either a existing service or one that is going to be created through the zgw-consumers setup step (and is declared in the same yaml file).

Reply:
I agree, I think this is going to be done after the initial deadline. This is something that all projects will have to mention in their docs, so we should think of an easy way to do this without repeating ourselves. Sergei mentioned that using intersphinx to link to the library documentation (with a section written for devops) could be an option

Reply:
Sounds good :+1: